src/iris/ui/static/js/iris.js

```diff
@@ -1548,6 +1552,30 @@ iris = {
         $this.prop('disabled', false);
       });
     },
+    resolveIncident: function(e){
+      var $this = $(e.target),
+          owner = $this.attr('data-action') === 'resolve' ? window.appData.user : null,
+          self = this,
+          incidentId = $this.attr('data-id');
+      //$this.prop('disabled', true);
+      $.ajax({
+        url: self.data.url + incidentId + '/resolve',
+        data: JSON.stringify({
+          owner: owner
+        }),
+        method: 'POST',
+        contentType: 'application/json'
+      }).done(function(){
+        self.getIncident(incidentId).done(function(){
+          var message = owner ? 'Incident ' + incidentId + ' marked as resolved.' : 'Incident ' + incidentId + ' marked as unresolved.';
+          iris.createAlert(message, 'success');
+        });
+      }).fail(function(){
+        iris.createAlert('Failed to modify incident', 'danger');
+      }).always(function(){
```

Comment:
Do we need this always callback? Looks like it only contains commented code.

Reply:
We do not, I just forgot to remove it.